gulpfile.js

```diff
@@ -49,6 +50,22 @@ gulp.task('watch', function () {
     gulp.watch('assets/css/**', ['css']);
 });
 
+gulp.task('build-dist', ['css'], function() {
+    var rimraf = require('rimraf');
+    var fs = require('fs');
+
+    var targetDir = 'dist/';
+    var themeName = JSON.parse(fs.readFileSync('./package.json')).name;
+    var filename = themeName + '.zip';
+
+    // remove the old dist/ directory
+    rimraf.sync(targetDir);
```

Comment:
This isn't needed - it will automatically overwrite 🙂

Reply:
You're right, but I left it in case someone decides to change the name of the theme in between builds. This shouldn't occur too often but I thought it wouldn't hurt to start from a "clean slate" and cover this case as well. But if you prefer to have it removed, that's fine for me too!

Reply:
My preference would be in favour of one-less-dependency. In this case a user can manually delete the file :)

Reply:
That's a good reason!